acloud/resource_cluster.go

```diff
@@ -329,8 +327,8 @@ func resourceClusterUpdate(ctx context.Context, d *schema.ResourceData, m interf
 	}
 
 	updateCluster := acloudapi.UpdateCluster{
-		UpdateChannel:               d.Get("update_channel").(string),
-		Version:                     d.Get("version").(string),
+		UpdateChannel:               d.Get("update_channel").(*string),
```

Comment:
I remember these pointer values cause some some issues.

Please ask @idebeijer

Reply:
Yes, I'd recommend checking if the value has a change with `d.Change` first and then setting it with `d.GetChange` like being done with other pointer values.